src/Common/src/Interop/OSX/Interop.RunLoop.cs

```diff
@@ -26,7 +27,11 @@ internal static partial class RunLoop
         /// <summary>
         /// Starts the current thread's RunLoop. If the RunLoop is already running, creates a new, nested, RunLoop in the same stack.
         /// </summary>
+#if MONO
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+#else
         [DllImport(Interop.Libraries.CoreFoundationLibrary)]
+#endif
```

Comment:
Is this change sufficient to make the Mono build of System.IO.FileSystem.Watcher happy? If yes, how is it dealing with platform-specific interop?